### DISCUSSION TITLE
取消libc6 ld.conf关于 /lib/x86_64-linux-gnu路径的声明讨论

### DISCUSSION THREAD

**Discussion by Zeno-sole** [linuxdeepin/developer-center#3833] [Developer Discussion | 开发者讨论]
debian等发行版为了保留兼容性(非usrmerge) 声明了/usr/lib 以及/lib路径，大多发行版默认都是提供usrmerge，debian12之后也不在计划支持非usrmerge状态，deepin一直依赖都是使用usrmerge，其实可以取消关于 /lib/x86_64-linux-gnu路径的声明

**Comment by Zeno-sole** [comment]
https://lists.debian.org/debian-devel-announce/2022/09/msg00001.html 
https://salsa.debian.org/md/usrmerge/raw/master/debian/README.Debian
https://wiki.debian.org/UsrMerge 
相关讨论以及发展历史请查阅链接

**Comment by Zeno-sole** [comment]
@Rabenda @zccrs @justforlxz

**Comment by justforlxz** [comment]
好事

**Comment by RevySR** [comment]
贴一下修改patch 以便进一步评估

以及相关usrmerge合并也会和debootstrap有一定关系

**Comment by black-desk** [comment]
但是实际上deepin的apt仓库里没有usr-is-merged这个包（x
